src/radio_tool_button.rs

```diff
@@ -0,0 +1,23 @@
+use std::ptr;
+
+use RadioToolButton;
+use ToolItem;
+use ffi;
+use glib::object::Downcast;
+use glib::translate::*;
+
+impl RadioToolButton {
+    pub fn new() -> RadioToolButton {
+        assert_initialized_main_thread!();
+        unsafe {
+            ToolItem::from_glib_none(ffi::gtk_radio_tool_button_new(ptr::null_mut())).downcast_unchecked()
+        }
+    }
+
+    pub fn new_from_stock(stock_id: &str) -> RadioToolButton {
+        assert_initialized_main_thread!();
+        unsafe {
+            ToolItem::from_glib_none(ffi::gtk_radio_tool_button_new_from_stock(ptr::null_mut(), stock_id.to_glib_none().0)).downcast_unchecked()
```

Comment:
This one is deprecated since 3.10, maybe let's not add it? Docs say that `gtk_radio_tool_button_new()` should be used instead.

Also it probably makes sense to expose *something* for the weird groups API it provides? IIRC we already have the same problem with other group API elsewhere in GTK ( @GuillaumeGomez you remember? ) so let's take inspiration from there.

Reply:
I don't recall which at all. :-/

Reply:
@EPashkin Maybe? Some other RadioWhatever API (the plain RadioButton? RadioMenuItems?) where you have groups of things. I don't have time for looking closer now and GTK UI stuff is not really my main area of expertise so I would have to look up, sorry :)

Reply:
You meant https://github.com/gtk-rs/gtk/issues/488?
Problem with this class that it don't have `join_group` function but it has `group` property that maybe works same and better deignored.
I will check this PR more later.

Reply:
I actually missed that the `from_stock` functions are deprecated. I can remove them if you want. I am fine either way.

For the group API, I will try to implement what @EPashkin proposed below. At the moment one has to use one of the `new_from_widget` functions.